### PR TITLE
Add readme template

### PR DESCRIPTION
**Pull request by TijsM**
## What

- add a readme template so all projects can use this one

inspired on: 
[https://github.com/bothrs/my-cancer-companion-app](https://github.com/bothrs/my-cancer-companion-app)

[https://github.com/bothrs/my-cancer-companion-backend](https://github.com/bothrs/my-cancer-companion-backend)

[https://github.com/bothrs/lets-march](https://github.com/bothrs/lets-march)

[https://github.com/bothrs/la-belle-maison-app](https://github.com/bothrs/la-belle-maison-app)

[https://github.com/bothrs/la-belle-maison-backend](https://github.com/bothrs/la-belle-maison-backend)

[https://github.com/bothrs/edison-backend](https://github.com/bothrs/edison-backend)

### CODE REVIEW

File: readme_template.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Link to Production
+
+- _Insert links to production (api, site, expo qr code, ...)_
+
+# General info
+
+## Linked Projects
```
Comment (bramvanhoutte) [inline readme_template.md]: Req: Write `Linked repositories`

File: readme_template.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| Authentication        | _link to auth provider, eg: auth0, firebase, ..._  |        |
+| figma                 | _link to Figma file_                               |        |
+
+## Used tools and accounts
+
+_list all tools, should already be in notion or Lastpass, so a link to this page will be enough_
+
+# Getting started
+
+## Secrets/ doppler
```
Comment (bramvanhoutte) [inline readme_template.md]: Req: Fix spacing

File: readme_template.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## How to run locally
+
+_Describe how to get the project up and running on you local machine_
+
+## Publishing
+
+_Describe how to publish the project to the different environments_
+
+# Code Documentation
```
Comment (bramvanhoutte) [inline readme_template.md:55]: Req: Consistent capitalization

**Review by bramvanhoutte — APPROVED** [review]

**Review by codinsonn — COMMENTED** [review]
Missing a "Tests" section?

**Comment by TijsM** [comment]
> Missing a "Tests" section?

What would you expect in a Test section? @codinsonn 

something like this?


# Getting started

## Tests
 
### Run tests

``` yarn run tests```

**Review by JaccoGoris — APPROVED** [review]

**Comment by TijsM** [comment]
> Missing a "Tests" section?

added it.

File: readme_template.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+_List the people who worked on this project_
+
+## General info
+
+| Item                  | Info / links                                       | Remark |
+| --------------------- | -------------------------------------------------- | ------ |
+| Framework             | _insert basic stack info, eg: "nest.js + graphql"_ |        |
+| Notion Track Hub link | _insert link to notion_                            |        |
+| Hosting provider      | _insert link to gcp/vercel/..._                    |        |
```
Comment (thgh) [inline readme_template.md]:
```suggestion
| Hosting provider      | _insert link to gcp/vercel/... project dashboard_                    |        |
```

File: readme_template.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+| Architecture map      | _link to architecture map in miro_                 |        |
```
Comment (thgh) [inline readme_template.md]:
```suggestion
| Architecture map      | _link to architecture map in Miro_                 |        |
```

File: readme_template.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+| Airtable Link         | _link to translations_                             |        |
+| Doppler               | _link to doppler_                                  |        |
```
Comment (thgh) [inline readme_template.md]:
```suggestion
| Doppler               | _link to Doppler_                                  |        |
```

File: readme_template.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+| Authentication        | _link to auth provider, eg: auth0, firebase, ..._  |        |
```
Comment (thgh) [inline readme_template.md]:
```suggestion
| Authentication        | _link to auth provider, eg: Auth0, Firebase, ..._  |        |
```

**Pull request merged**